pkg/crypto/internal/wallet/wallet.go

```diff
@@ -37,15 +40,18 @@ type provider interface {
 	InboundTransportEndpoint() string
 }
 
-// BaseWallet wallet implementation
-type BaseWallet struct {
+// SecretWallet wallet implementation, including access to private keys.
+// Note: this is not exposed to the rest of the library, only to pkg/crypto.
+// Other packages need to use pkg/crypto/wallet, which wraps this,
+//   hiding the access to private keys.
+type SecretWallet struct {
```

Comment:
Why is did store and inboundTransportEndpoint part of "SecretWallet".

(the secret part is really the keystore and the whole "wallet" name could be dropped).

Reply:
Instead of answering the question I'm going to agree